docs/user/openstack/README.md

```diff
@@ -176,68 +216,132 @@ Create and associate a floating IP to the ingress port:
 openstack floating ip create --port <ingress port id> <external network>
 ```
 
-Add A record in your dns for *apps. in your DNS:
-
-```
-*.apps.example.shiftstack.com  IN  A  <ingress FIP>
-```
-OR add A record in `/etc/hosts`:
-
-```
-    <ingress FIP> console-openshift-console.apps.example.shiftstack.com
-```
-
 ### Without Floating IPs
 
 If you don't want to pre-create a floating IP address, you will still want to create the API DNS record or the installer will fail waiting for the API.
```

Comment:
I'm not sure how this will work. What do you set the API address to? You won't be able to get to the private `.5` AIP VIP address...